flow/size.yaml

```diff
@@ -62,3 +83,107 @@ components:
           type: integer
           default: 1518
           x-field-uid: 2
+    Flow.SizeWeightPairs:
+      type: object
+      description: |-
+        Custom frame size distribution, defined as <size, weight> pairs (including IMIX distribution).
+      properties:
+        choice:
+          type: string
+          default: predefined
+          x-field-uid: 1
+          x-enum:
+            predefined:
+              x-field-uid: 1
+            custom:
+              x-field-uid: 2
+        predefined:
+          type: string
+          default: imix
+          description: |-
+            Option allow to specify a predefined frame size distribution <size, weight> pairs (including IMIX distribution).  
+            The available predefined distribution pairs are:
```

Comment:
Is there a source document for these pre-defined distributions if someone wants to check details or check if more have been added in future ?

Reply:
Didn't find any RFC, but these mix are proposed by different test vendors (e.g. [ixia ref doc](http://www.ixiacom.com/pdfs/test_plans/agilent_journal_of_internet_test_methodologies.pdf) ).
All pre-defined options proposed in this model are based on IxNetwork.